modules/Settings/loaddata.php

```diff
@@ -0,0 +1,174 @@
+<?php
+$Vtiger_Utils_Log = true;
+include_once('vtlib/Vtiger/Module.php');
+global $current_user,$adb;
+set_time_limit(0);
+ini_set('memory_limit','1024M');
+$fldname = $_GET['empid'];
+?>
+
+
+<?php
+
+// Workflow Conditions
+       $crs = $adb->query("SELECT workflow_id,summary
+                FROM `com_vtiger_workflows`
+                WHERE test like '%$fldname%'");
+            if ($crs and $adb->num_rows($crs)>0) {
```

Comment:
use pquery here to avoid sql injection

```
$crs = $adb->pquery('SELECT workflow_id,summary FROM `com_vtiger_workflows` WHERE test like ?', array('%'.$fldname.'%');
```